docs/source/notebooks/power_analysis.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>Can you just double check this... when I ran nbqa-ruff it removed some unnecessary imports, so we might not need to mention some of the packages that you mention here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/277/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>